data/modules/Assassination/Assassination.lua

```diff
@@ -99,8 +99,8 @@ local onChat = function (form, ref, option)
 		local sbody = ad.location:GetSystemBody()
 
 		form:SetMessage(string.interp(l.X_WILL_BE_LEAVING, {
-		  target    = ad.target,
-		  spaceport = sbody.name,
+		  target    = true and ad.targetsurname or ad.target,	-- If targetsurname is missing, then older version.
```

Comment:
Note that you do not need a trinary expression here - `ad.targetsurname or ad.target` would suffice.

Reply:
Fixed!